client/packages/invoices/src/InboundShipment/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -219,6 +223,16 @@ const useStatusChangeButton = () => {
   };
 };
 
+export const validateEmptyInvoice = (lines: {
+  totalCount: number;
+  nodes: InboundLineFragment[];
+}): boolean => {
+  // Should only proceed if there is at least one line, with received or shipped packs defined
+  if (lines.totalCount === 0 || lines.nodes.every(isInboundPlaceholderRow))
```

Comment:
> // Should only proceed if there is at least one line, with received or shipped packs defined

If it's an old inbound shipment that we want to zero out (what the table action I believe is for), we should be able to verify it when all lines are zero (I think), would this prohibit it ? If so can we clarify with @mark-prins

Reply:
!row.shippedNumberOfPacks;